### PR TITLE
Feature/realm ios backfill

### PR DESCRIPTION
**Pull request by troach-sf**
#### Description:

Adds backfilling to iOS location storage, bringing in sync with Android

#### Linked issues:

Completion of work: https://github.com/Path-Check/covid-safe-paths/pull/788

#### How to test:

Run native iOS test suite in `COVIDSafePathsTests` scheme

### CODE REVIEW

File: ios/COVIDSafePaths.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -708,11 +702,11 @@
 			children = (
 				B2A58213244CE76C000F39A6 /* en */,
 				B2A58216244CE778000F39A6 /* it */,
-				B2A5821E244CE988000F39A6 /* fr */,
+				B2A5821E244CE988000F39A6 /* sk */,
 				B2A58220244CE9EB000F39A6 /* ro */,
 				B2A58222244CE9F7000F39A6 /* id */,
 				B2A58224244CEA20000F39A6 /* fr-HT */,
-				B2A58226244CEA6C000F39A6 /* ml */,
+				B2A58226244CEA6C000F39A6 /* pt-BR */,
```
Comment (tstirrat) [inline ios/COVIDSafePaths.xcodeproj/project.pbxproj:709]: We lost/changed a couple of languages. 

fr is gone, ml is gone.

Can you revert the language related changes in this file?

Comment (troach-sf) [reply ios/COVIDSafePaths.xcodeproj/project.pbxproj:709]: @tstirrat As we've discussed in Slack, these changes were automatically completed by Xcode due to those string translations being empty files. I'd hesitate to modify this file directly going against Xcode's auto-modify. I'm not 100% sure what we are missing by not having these translations listed here. The fallback would always be to show in a default language. I don't think I've got time to look much further into it. Can you or someone else take a further look?

Comment (tstirrat) [reply ios/COVIDSafePaths.xcodeproj/project.pbxproj:709]: We can take that offline. As long as ft-HT is still there its not blocking prod builds.

**Comment by tstirrat** [comment]
Verified that these tests pass. There is one failing test that predates this change.

**Comment by tstirrat** [comment]
```
Test Suite 'COVIDSafePathsTests' failed at 2020-05-12 13:42:31.701.
	 Executed 1 test, with 2 failures (0 unexpected) in 1.298 (1.298) seconds
Test Suite 'LocationTests' started at 2020-05-12 13:42:31.701
Test Case '-[COVIDSafePathsTests.LocationTests testNewYorkAndSydneyNotNearby]' started.
Test Case '-[COVIDSafePathsTests.LocationTests testNewYorkAndSydneyNotNearby]' passed (0.001 seconds).
Test Case '-[COVIDSafePathsTests.LocationTests testNorthAndSouthPolesNotNearby]' started.
Test Case '-[COVIDSafePathsTests.LocationTests testNorthAndSouthPolesNotNearby]' passed (0.000 seconds).
Test Case '-[COVIDSafePathsTests.LocationTests testSpotsInKansasCityAreNearby]' started.
Test Case '-[COVIDSafePathsTests.LocationTests testSpotsInKansasCityAreNearby]' passed (0.000 seconds).
Test Suite 'LocationTests' passed at 2020-05-12 13:42:31.704.
	 Executed 3 tests, with 0 failures (0 unexpected) in 0.001 (0.002) seconds
Test Suite 'RealmSecureStorageTest' started at 2020-05-12 13:42:31.704
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testAssumedLocationListEmptyIfNotNearby]' started.
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testAssumedLocationListEmptyIfNotNearby]' passed (0.006 seconds).
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testAssumedLocationListGenerated]' started.
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testAssumedLocationListGenerated]' passed (0.005 seconds).
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testGetLocationsReturnsSortedAndFilters]' started.
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testGetLocationsReturnsSortedAndFilters]' passed (0.012 seconds).
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testImportFailsIfLocationOlderThanMinimum]' started.
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testImportFailsIfLocationOlderThanMinimum]' passed (0.008 seconds).
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testImportSuccessFromGoogle]' started.
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testImportSuccessFromGoogle]' passed (0.007 seconds).
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testImportSuccessFromMigration]' started.
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testImportSuccessFromMigration]' passed (0.007 seconds).
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testMaxAssumedLocationListEmptyIfTimeTooClose]' started.
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testMaxAssumedLocationListEmptyIfTimeTooClose]' passed (0.004 seconds).
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testMaxAssumedLocationListGenerated]' started.
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testMaxAssumedLocationListGenerated]' passed (0.005 seconds).
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testOldLocationsTrimmed]' started.
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testOldLocationsTrimmed]' passed (0.010 seconds).
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testSaveDeviceLocation]' started.
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testSaveDeviceLocation]' passed (0.007 seconds).
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testSaveDeviceLocationIgnoredIfPreviousInsertTooClose]' started.
Test Case '-[COVIDSafePathsTests.RealmSecureStorageTest testSaveDeviceLocationIgnoredIfPreviousInsertTooClose]' passed (0.009 seconds).
Test Suite 'RealmSecureStorageTest' passed at 2020-05-12 13:42:31.787.
```

failing test:

```
Test Case '-[COVIDSafePathsTests testRendersWelcomeScreen]' failed (1.298 seconds).
```

**Pull request merged**